### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#429] [Esami & Esoneri M-Z]
## Traccia

<img width="923" alt="image" src="https://github.com/user-attachments/assets/277973ed-1c46-4631-8edd-f721f04daba7">

**Comment by rimaout** [comment]
<img width="1200" alt="image" src="https://github.com/user-attachments/assets/98bb5efa-f0bf-4849-8ccf-8bbf27c52a6c">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/86a7203c-d2af-4f18-a4a1-080b73ab0f88)
